chips/sam4l/src/lib.rs

```diff
@@ -7,6 +7,7 @@
 #![feature(attr_literals, const_cell_new)]
 #![feature(const_atomic_usize_new, const_ptr_null_mut, integer_atomics, try_from)]
 #![feature(asm, core_intrinsics, concat_idents, const_fn)]
+#![feature(used)]
```

Comment:
The others above appended to the existing feature list, better to be consistent?

Reply:
I mean, it has been pretty arbitrary where these go. Do you really care?

Reply:
No, I approved the changes. Just was pointing it out.

Realistically, it's something I imagine rustfmt will do something with someday.